### PR TITLE
Migrate docs docs.datajoint.org

### PR DESCRIPTION
**Pull request by CBroz1**
Before merge, we should
1. Merge #17 (hard wrapping of existing materials)
2. Decide the merge order of this PR vs. #14 - After discussion, I'll pull commits accordingly

For this PR, I ...
1. Collected all docs from docs.datajoint.org and all snippets from datajoint-python and datajoint-matlab
2. Converted previous .rst docs into markdown, including tabbing markers for python- vs. matlab-specific content
3. Moved all docs into an archive folder in this repo, which includes the following subfolders
    - `_planned_archive`: I believe this material should be archived as we migrate to more user-friendly language
    - `_to_python_matlab`: These are the materials that remain after I took out general information. In future PRs, I plan to delete the contents here and load into their respective docs sits. Maybe next week?
    - `_to_include`: Remaining docs are here. These are candidates for inclusion that could be pulled from to continue adding docs in the future.
4. Added a number of pages, revising existing content to be more new-user friendly
5. Generated updated figures and mermaid diagrams to fit updated branding. Some of these came from google sheets/slides - where on google drive should they be stored?

### CODE REVIEW

File: src/concepts/ref-integrity/relational-databases.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Relational Databases
+
+Coming soon!
+
+While you wait, you may be interested in our 
+[previous documentation](https://docs.datajoint.org/python/admin/2-configuration.html)
```
Comment (kabilar) [inline src/concepts/ref-integrity/relational-databases.md:6]:
```suggestion
```

Comment (kabilar) [reply src/concepts/ref-integrity/relational-databases.md:6]: Redirects will be happening this week.

**Review by dimitri-yatsenko — DISMISSED** [review]

File: src/community/support.md [hunk 2]
```diff
@@ -12,10 +12,17 @@ We have a general policy for DataJoint open-source [community engagement](
...
+    question.  Teams who wish to receive this support can either share their workflow
+    directly or work with DataJoint engineers to replicate the issue in a test
+    environment.   To be fair to the community, no team will be selected for support
+    more than once per quarter.
 
   + [Sign up for Office Hours!](https://docs.google.com/forms/d/e/1FAIpQLSeMhZtzQQWB47I8HfPcJ5_pFyMhZO284PLIblDfshe30dEuXw/viewform)
 
-For fully managed services, we invite you to visit the [DataJoint Solutions](https://www.datajoint.com/solutions) page and request more information.
+For fully managed services, we invite you to visit the 
+[DataJoint Solutions](https://www.datajoint.com/solutions) page and request more information.
```
Comment (kabilar) [inline src/community/support.md:28]:
```suggestion
For fully managed services, we invite you to visit 
[DataJoint](https://www.datajoint.com) for solutions and request more information.
```

**Review by kabilar — APPROVED** [review]
Thanks @CBroz1!  Huge effort here on this migration.

I did not review all the text in detail, but our team will collectively review and update over the next 1-2 months.

Let's merge #14 after this pull request.  We can keep the the Google Slides/Sheets in the U24 folder for now, and find a way to add those files to this repo in the future.

**Pull request merged**